models/lease.py

```diff
@@ -24,3 +25,8 @@ def json(self):
             "dateTimeEnd": Time.format_date(self.dateTimeEnd),
             "unitNum": self.unitNum,
         }
+
+    @classmethod
+    def active(cls):
+        time = now()
+        return (time > cls.dateTimeStart) & (time < cls.dateTimeEnd)
```

Comment:
Re: the bitwise &: I originally tried `and`. It turns out that won't work, because SQLalchemy can't override python's `and`! The & being used here is not actually Python's bitwise and -- it's SQLalchemy's overridden "and" operator. I could also change this to use the equivalent, explicit `and` function that exists somewhere in SQLalchemy.

Re: strings vs datetimes: I actually don't know 😶 I will have to track down the exact datatype of the column involved. I generally work on Dwellingly work on Wednesdays, so I'll do that then.

Reply:
Huh! I had no idea SQLalchemy did that! In that case, I think it's totally fine to use `&` here.
You learn something new every day...

Reply:
I had no idea either. Thanks for explaining this to us @gussmith23